my-app/src/components/Firebase/firebase.js

```diff
@@ -1,16 +1,26 @@
-import app from 'firebase/app';
+//import firebase from 'firebase/app'
+import  'firebase/firestore'
+import * as firebase from 'firebase'
+import 'firebase/storage'
 
-const config = {
-  apiKey: process.env.REACT_APP_API_KEY,
-  authDomain: process.env.REACT_APP_AUTH_DOMAIN,
-  databaseURL: process.env.REACT_APP_DATABASE_URL,
-  projectId: process.env.REACT_APP_PROJECT_ID,
-  storageBucket: process.env.REACT_APP_STORAGE_BUCKET,
-  messagingSenderId: process.env.REACT_APP_MESSAGING_SENDER_ID,
-};
-class Firebase {
-  constructor() {
-    app.initializeApp(config);
-  }
-}
-export default Firebase;
\ No newline at end of file
+  var firebaseConfig = {
+    apiKey: "AIzaSyCbRkA5K0ZtmftnVW4Q8TVb5t6MHCzGg2Y",
```

Comment:
Please move it to `process.env....` and regenerate credentials.
PS did you find a problem why `process.env...` didn't worked?

Reply:
"@firebase/firestore: Firestore (7.13.2): Could not reach Cloud Firestore backend."

unfortunatelly, not

Reply:
looks like some bad credentals were taken --> then we'll take a look at it